src/pages/builders/challenges.tsx

```diff
@@ -1,22 +1,81 @@
+import { useState } from 'react'
+import type { GetServerSideProps, NextPage } from 'next'
+import classNames from 'classnames'
 import { config } from '@/lib/config'
+import Phases from '@/data/phase'
 import { Head } from '@/components/head/Head'
 import { Header } from '@/components/layout/header/Header'
 import { Footer } from '@/components/layout/footer/Footer'
-import type { GetServerSideProps, NextPage } from 'next'
 import type { Config } from '@/types/config.type'
+import type { ChallengeDTO } from '@/data/phase/dto.type'
 
-export type ChallengesProps = Pick<Config, 'title' | 'keywords' | 'description' | 'urls'>
+export type ChallengesProps = Pick<Config, 'title' | 'keywords' | 'description' | 'urls' | 'phases'>
+
+type CategoryTabProps = {
+  toggleTab: () => void
+  selectedTab: ChallengeDTO
+  challenge: ChallengeDTO
+}
+
+const CategoryTab: React.FC<CategoryTabProps> = ({ toggleTab, challenge, selectedTab }) => (
+  <li
+    className={classNames('okp4-nemeton-web-page-challenges-categories-tab', {
+      active: challenge === selectedTab
+    })}
+    onClick={toggleTab}
+  >
+    {challenge.challengeCategorie}
+  </li>
+)
 
 const Challenges: NextPage<ChallengesProps> = props => {
-  const { urls } = props
+  const { urls, phases } = props
+  const [categorieChallenges] = useState<ChallengeDTO[]>(
+    Phases(phases)[Phases(phases).length - 1].challenges ?? []
+  )
+  const [selectedCategorie, setSelectedCategorie] = useState<ChallengeDTO>(categorieChallenges[0])
+
+  const toggleCategorie = (tab: ChallengeDTO) => () => setSelectedCategorie(tab)
 
   return (
     <div className="okp4-nemeton-web-page-main">
       <Head {...props} />
       <main>
         <Header />
-        <div className="okp4-nemeton-web-page-content-container" id="challenges">
-          <h1>Challenges</h1>
+        <div className="okp4-nemeton-web-page-content-container challenges" id="challenges">
+          <div className="okp4-nemeton-web-page-challenges-container">
+            <div className="okp4-nemeton-web-page-challenges-content">
+              <h1>Challenges</h1>
+              <div className="okp4-nemeton-web-page-divider" />
+              <h2>part 1: Invoke the power of Dagda - From May, 24th</h2>
+              <p>
+                In the Micro challenges phase, we&apos;ll be invoking the power of Dagda, one of the
+                main gods of Celtic mythology, to develop specific bricks to interact with the
+                protocol. Indeed, before having complete and complex applications, it is essential
+                to have a panel of services available, almost elementary bricks, which will
+                stimulate the creativity of developers.
+              </p>
```

Comment:
This should be part of the data carried by the phase, as done for the tasks, not hardcoded in the component...